plugins/chat/assets/javascripts/discourse/controllers/preferences-chat.js

```diff
@@ -99,6 +114,13 @@ export default class PreferencesChatController extends Controller {
     return this.model.get("user_option.chat_send_shortcut");
   }
 
+  get chatQuickReactionsCustom() {
+    let emojis =
```

Comment:
```suggestion
    const emojis =
```

minor, but you aren't modifying this